scripts/extension_helpers.js

```diff
@@ -2,6 +2,20 @@
 // Distributed under the terms of the Modified BSD License.
 
 var path = require('path');
+var findImports = require('find-imports');
+
+// Get the list of vendor files.
+var VENDOR_FILES = findImports('../lib/**/*.js', { flatten: true });
+var CODEMIRROR_FILES = VENDOR_FILES.filter(function(importPath) {
+  return importPath.indexOf('codemirror') !== -1;
```

Comment:
Is it enough to check if a string *starts* with codemirror? That way if we end up with a codemirror directory in the future (for example, the notebook has a codemirror directory now), we only pull in the actual codemirror package.